README.md

```diff
@@ -1 +1,70 @@
-# catalog-of-my-things
\ No newline at end of file
+# Catalog of our things
+Using OOP, we created a catalog where the user can add books, games and music albuns, having as properties genre, author, and label. It is possible to display a list of all elements by the UI displayed on the console. We followed the following UML diagram structure to also create a schema.sql:
+
+![image](https://user-images.githubusercontent.com/42799579/188950909-66f7bb00-dc45-47e8-8457-cf5669716c8d.png)
+
+## Built With:
+
+- Ruby
+- SQL
+
+## Video Presentation:
+[Check our video presentation about the project](https://drive.google.com/file/d/1JNt88GczT7MJ81UHPq71Oq4qXjrkJk5_/view).

```

Comment:
- [ ] [REQUIRED] I couldn't see your project online, I don't know why, but I tried downloading it and watching it. You gave good information about the project, but you didn't mention the part of the code in this project that you are proud of, which `Alex` wanted to talk about, but he didn't have enough time. Please, it will be enough for each of you to give information about the project in `1 minute`, and in the remaining `2 minutes`, give open and complete information about the part I requested, it is very important for you.

Reply:
Ok, thanks for the review.

Reply:
Well resolved 👍🏽 
Kindly add the additional presentation to the README file also. 
https://drive.google.com/drive/folders/1pPjRwBzThvzkzeN4mxfZzEDpewoyRbAa?usp=sharing